server/utils/assessments/dateUtils.ts

```diff
@@ -38,6 +38,17 @@ const daysUntilDue = (assessment: AssessmentSummary): number => {
   return differenceInDays(dueDate, new Date())
 }
 
+const daysToWeeksAndDays = (days: string | number): { days: number; weeks: number } => {
+  const daysAsNumber = Number(days)
+  // eslint-disable-next-line no-bitwise
+  const durationWeeks = (daysAsNumber / 7) | 0
```

Comment:
this should be `(daysAsNumber / 7) || 0`. We're using the bitwise operator here which I think is unintentional so we can remove line 43 too

Reply:
We do need the bitwise operator for this, but I'm not sure _why_ it works! I've gone for `const durationWeeks = Math.floor(daysAsNumber / 7)` as it's more readable